modules/deploy/pages/deployment-option/cloud/aws-privatelink.adoc

```diff
@@ -0,0 +1,403 @@
+= Configure AWS PrivateLink for Redpanda Cloud
+:description: Set up AWS PrivateLink to securely access Redpanda Cloud. 
+:page-cloud: true
+
+include::shared:partial$feature-flag.adoc[]
```

Comment:
@bpraseed This banner will render like so:
<img width="973" alt="image" src="https://github.com/redpanda-data/docs/assets/36839689/e101ef8e-adc2-4603-8ad1-78989e0e3c13">
Is this OK or is there a preferred message for this feature?

Reply:
LGTM